include/libdnf/rpm/nevra.hpp

```diff
@@ -79,6 +85,8 @@ struct Nevra {
     std::string arch;
 };
 
+inline const std::vector<Nevra::Form> Nevra::FORMS_MOST_SPEC ({Nevra::Form::NEVRA, Nevra::Form::NA, Nevra::Form::NAME, Nevra::Form::NEVR, Nevra::Form::NEV});
```

Comment:
`FORMS_MOST_SPEC` is difficult to understand to me.
Wouldn't be better to change it to something like `FORMS_DEFAULT` or `DEFAULT_FORMS`?